src/common/Messages.h

```diff
@@ -103,4 +103,11 @@ enum TxSharingMode : unsigned char
     SEND_AND_FORWARD = 0x04
 };
 
+enum HeaderStartByte : unsigned char
```

Comment:
- convert to `enum class`
- confirm that this a good place to define it 
- confirm that there are no more possible start bytes for headers